### PR TITLE
Upgrade devel branch to Fedora 28

### PR DESCRIPTION
**Pull request by Tjzabel**
## What This PR Does
* **Moves official working changes from my [GitLab fork](https://gitlab.com/Tjzabel/TigerOS) into TigerOS main project**
* Sets TigerOS to F28
* Puts up to date with [latest devel ISO](https://mirrors.ritlug.com/TigerOS/28/release/)
* Adds miscellaneous screenshots and improvements

### CODE REVIEW

File: kickstarts/pkgs-repos.ks
Comment (Tjzabel) [inline kickstarts/pkgs-repos.ks]: Change this back to F28 using $releasever

File: kickstarts/tigeros-source.ks
Comment (Tjzabel) [inline kickstarts/tigeros-source.ks]: Change this back to F28 using $releasever

(Was pretty sure I had done this)

**Review by Tjzabel — COMMENTED** [review]
Need to update a few repos to $releasever

**Comment by Tjzabel** [comment]
I need to rebase my commits before the merge, so please only approve the review, and let me handle the merging

**Review by linux-modder — APPROVED** [review]
Looks good overall, mentioned some redundancy in commits via irc but in review those are non blocking and easily fixed via merge|rebase  --cherry:

- Make a builddir |resultsdir in makeiso.sh so that it is extensible and allows folks on restictive|ed systems to still create isos 
- Remove a few redundant commits in the history (rebase --squash --interactive)

File: make-iso.sh
Comment (Tjzabel) [inline make-iso.sh]: Maybe think of different location for the build to occur, such as `/var/lmc/foo` instead of what's currently there. It may not be a good idea to delete that directory for _other_ developers who wish to use this file to build a personal ISO

File: kickstarts/tigeros.ks
Comment (Tjzabel) [inline kickstarts/tigeros.ks]: Remove repkg RPMs, and only install them through the postinstall script. 
These can potentially bring in legal disputes if installed by default

File: make-iso.sh
Comment (Tjzabel) [inline make-iso.sh]: update `/var/lmc` to a subset directory. Maybe have `/var/lmc/TigerOS-${date}` or something similar

Comment (axk4545) [reply make-iso.sh]: You didn't actually update it... you need to change the `--resultdir` arg

Comment (Tjzabel) [reply make-iso.sh]: @axk4545 I didn't make any changes yet :smile:

**Review by axk4545 — COMMENTED** [review]
I will leave final approval to @ct-martin but it looks like good progress to me. @Tjzabel I would look into whether LMC relies on using those directories before changing them. I also agree the the addition of `$basearch` is a good idea but would like to point out that AFAIK we only have 64 bit packages except for a couple noarch

File: make-iso.sh
Comment (axk4545) [reply make-iso.sh]: I also just noticed my typo now after what? ... a year

**Comment by Tjzabel** [comment]
@ct-martin I've updated the result_dir location to `/var/lmc/TigerOS` for the time being. Let me know if you'd rather have multiple result directories.

**Comment by Tjzabel** [comment]
@ct-martin let me know how you like the current changes.

**Comment by Tjzabel** [comment]
Due to this PR being up for quite a while with no visible progress, and adding the necessary commits needed for approval for @ct-martin and @linux-modder, I am going to merge this PR. Further additions can be made from another PR.

**Pull request merged**
